### PR TITLE
Test/get username

### LINKED ISSUE #128: Test getUsername()
**Description**
- Tests for getUsername()

**Requirements**
- [x] Write the necessary tests for the getUsername() utility funcion
- [x] Make sure getUsername() passes all the tests

### PR DESCRIPTION
**Pull request by OUMontiel**
# What does this PR do?
- Tests getUsername() function

**Related Issues**
- Closes #128 

---
**Merge Checklist**
- [x] Branch is up to date
- [x] No code style issues

### CODE REVIEW

File: src/test/java/com/google/sps/UtilityTest.java [hunk 1]
```diff
@@ -16,12 +16,15 @@
 
 import com.google.sps.classes.SubjectTag;
 import com.google.sps.classes.Utility;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
```
Comment (abundis28) [inline src/test/java/com/google/sps/UtilityTest.java:20]: I guess there is no use for the response import.

Comment (OUMontiel) [reply src/test/java/com/google/sps/UtilityTest.java]: Removed HttpServletResponse import!

File: src/test/java/com/google/sps/UtilityTest.java [hunk 2]
```diff
@@ -115,4 +117,53 @@ public void nullValue() {
...
+    int userId = 1;
+    
+    String actual = Utility.getUsername(userId, request);
+    String expected = "shaargtz";
+
+    Assert.assertEquals(actual, expected);
+  }
+
+  @Test
+  public void validIdWithNoUser() {
```
Comment (gloriapark) [inline src/test/java/com/google/sps/UtilityTest.java:135]: A common style practice for naming java unit test functions it to include the method being tested, the test condition, and the expected outcome, so something like: getUsername_validIdWithNoUser_returnsEmptyString()

Doing this makes it really explicitly clear what is being tested, how it's being tested, and what we expect to see. Consider updating your methods to also follow this pattern!

Comment (gloriapark) [reply src/test/java/com/google/sps/UtilityTest.java:135]: It's not technically in the public github Java style guide, so I'll leave it up to you - but it's a good habit to get in :)

Comment (OUMontiel) [reply src/test/java/com/google/sps/UtilityTest.java]: Updated them all!

**Pull request merged**